safep/fileIO.py

```diff
@@ -128,4 +128,136 @@ def read_Files(files, step=1):
     data.index = data.window
     data["dVdW"] = data.vdW_ldl - data.vdW_l
     
-    return data
\ No newline at end of file
+    return data
+
+
+def parse_Colvars_log(filename):
+    '''
+    Parses Colvars standard output from a log file
+
+    Returns dictionaries containing key-value pairs
+    global_conf: top-level Colvars parameters
+    colvars: list of dicts, one for each colvar
+    biases: list of dicts, one for each bias
+
+    Note: within colvars, parameters of sub-objects CVCs and atom groups are found as nested dictionaries
```

Comment:
What's a CVC?

Reply:
It's Colvars lingo. Collective variable component, they are the pre-implemented basis functions. Giacomo and I coined that phrase at the lab's round table in the UPenn Chemistry Department's first floor, back in our young days.

Reply:
Replacing for clarity.